src/NLog/Config/XmlLoggingConfiguration.cs

```diff
@@ -847,12 +847,13 @@ private void ParseExtensionsElement(NLogXmlElement extensionsElement, string bas
                     }
                     catch (Exception exception)
                     {
+                        InternalLogger.Error(exception, "Error loading extensions.");
+
                         if (exception.MustBeRethrown())
                         {
                             throw;
                         }
-
-                        InternalLogger.Error("Error loading extensions: {0}", exception);
+                        
                         if (LogManager.ThrowExceptions)
```

Comment:
This does not preserve the old logic: This `if` branch will never run because if `LogManager.ThrowExceptions` is `true`, then `MustBeRethrown()` above would be `true`.